libopenage/renderer/demo/demo_5.cpp

```diff
@@ -38,6 +39,7 @@ void renderer_demo_5(const util::Path &path) {
 		camera->resize(w, h);
 	});
 
+	camera::CameraManager camera_manager(camera);
```

Comment:
The demos in this file should probably not use the camera manager because that introduces more unnecessary complexity to them. We try to show off specific features of the renderer in the code. As `CameraManager` is an interface between the input system and the renderer, it is technically an additional layer that does a bunch of updates which are unnecessary for the demo.